bandit/plugins/general_hardcoded_password.py

```diff
@@ -15,14 +15,13 @@
 # under the License.
 
 import ast
+import re
 import sys
 
 import bandit
 from bandit.core import test_properties as test
 
-
-CANDIDATES = set(["password", "pass", "passwd", "pwd", "secret", "token",
-                  "secrete"])
+RE_CANDIDATES = re.compile('pass|pwd|secret|token', re.IGNORECASE)
```

Comment:
I like the `re.IGNORECASE`. However I think this regex will match too many unwanted variable names e.g. "underpass", "tokeniser", "passionate".

I'd prefer a regex something like `r"(^|_)((pas+wo?r?d|pass|pwd|token)$|(secrete?)(_|$))"` which matches:

- *_passwd
- my_password
- passwrd
- *_token
- AUTH_TOKEN
- secret_* 
- secret_code

Reply:
Thanks for your comments are very appreciated.
I just fix it with a new regular expression, please let me know if do you like it.